examples/seismic/viscoacoustic/operators.py

```diff
@@ -624,13 +633,25 @@ def BornOperator(model, geometry, space_order=4, kernel='sls', time_order=2, **k
                       space_order=space_order, staggered=NODE)
     dm = Function(name='dm', grid=model.grid, space_order=0)
 
+    if time_order == 1:
+        v = VectorTimeFunction(name="v", grid=model.grid,
+                               time_order=time_order, space_order=space_order)
+        kwargs.update({'v': v})
+
     # Equations kernels
     eq_kernel = kernels[kernel]
     eqn1 = eq_kernel(model, geometry, p, r=rp, **kwargs)
 
     s = model.grid.stepping_dim.spacing
 
-    q = -dm * (p.forward - 2 * p + p.backward) / (s**2)
+    if time_order == 1:
+        dv = VectorTimeFunction(name="dv", grid=model.grid,
+                                time_order=time_order, space_order=space_order)
+        kwargs.update({'v': dv})
```

Comment:
'v' or 'dv' ?

Reply:
The method `sls_1st_order(model, geometry, p, r=None, **kwargs) `search for the particle velocity by:

```
# Particle Velocity
  v = kwargs.pop('v')
```

However, for the Born Operator, we need to invoke the `sls_1st_order(model, geometry, p, r=None, **kwargs)` method twice:

The first one call the method inserting `v`.

The second call the method inserting `dv.`

So, for the method to remain generic, we use` kwargs.update({'v': dv}) `as a solution